web/client/plugins/styleeditor/StyleCodeEditor.jsx

```diff
@@ -198,11 +198,11 @@ function StyleEditor({
                     <Toolbar
                         buttons={[
                             {
-                                className: 'square-button-md no-border',
-                                glyph: 'code',
-                                active: editorType === 'textarea',
+                                className: 'square-button-xs no-border',
+                                bsSize: 'xs',
                                 disabled: loading,
-                                tooltipId: editorType === 'visual'
+                                active: editorType === 'textarea',
+                                textId: editorType === 'visual'
                                     ? 'styleeditor.switchToTextareaEditor'
                                     : 'styleeditor.switchToVisualEditor',
```

Comment:
The request from the original issue is to use `Visual editor` and `Code editor` as labels https://github.com/GeoNode/geonode-mapstore-client/issues/230 while these ids are:
https://github.com/geosolutions-it/MapStore2/blob/94ce9a5d3126cd8ab4e18a4ef75411b4a907ac3f/web/client/translations/data.en-US.json#L2226-L2227

Could you please add the translations as suggested in the issue?